src/main/java/org/vanilladb/core/storage/metadata/TableMgr.java

```diff
@@ -172,6 +176,17 @@ public void dropTable(String tblName, Transaction tx) {
 				fcatfile.delete();
 		}
 		fcatfile.close();
+
+		// remove corresponding indices
+		Map<String, IndexInfo> tblindex = VanillaDb.catalogMgr().getIndexInfo(tblName, tx);
```

Comment:
We usually use lower camel case for variable names. Although that is not necessary, we will appreciate that you make them lower camel case.